docs/post-install.md

```diff
@@ -0,0 +1,61 @@
+Post-install
+------------
+
+**Important installer messages**
+
+At the end of the installation process, the installer will display messages like:
+```
+==> valkyrie: TASK: [getvalkyrie.valkyrie | Output the deploy key] ************************** 
+==> valkyrie: ok: [localhost] => {
+==> valkyrie:     "var": {
+==> valkyrie:         "ssh_public_key_output.stdout_lines": [
+==> valkyrie:             "/////////////////////////////////////////////////////////////////////////////",
+==> valkyrie:             "  Add the following deploy key to your project on Github:",
+==> valkyrie:             "=============================================================================",
+==> valkyrie:             "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCosL1qmesmA55S2ZB3fEdsLQSElHOARcXpu7ZeT5vX+kpKb6OQQ/6zQ9I5nx1GaZORkwlKy4qNxhmUiy7u/aIiPALQlL2m9y7brqFxewPMCcX5vhV84rW/qxieJzx3nSC0s99ui+vnYhylbwBapJmfpUcuCMJzyP+JlusRYTLyk/uIrDtewnk2PgAtFYF/gUyK6It/RIbwdtawXrW/MQuFuYPpVYnvG+bbqmx+t7HBISWBbm/DK2guKhXzkUhdD24sdPp9tjUfLfsWYB3Plh5mhOM8I4cTwjVOxGwVfwC+KiEAxlRbSihsqUaRdIr7AXfZ4pG/b6MelwlcX3W61tvz Valkyrie deploy key",
+==> valkyrie:             "=============================================================================",
+==> valkyrie:             "/////////////////////////////////////////////////////////////////////////////"
+==> valkyrie:         ]
+==> valkyrie:     }
+==> valkyrie: }
+==> valkyrie: 
+==> valkyrie: TASK: [getvalkyrie.valkyrie | Output the login link] ************************** 
+==> valkyrie: ok: [localhost] => {
+==> valkyrie:     "var": {
+==> valkyrie:         "aegir_login_link_output.stdout_lines": [
+==> valkyrie:             "/////////////////////////////////////////////////////////////////////////////",
+==> valkyrie:             "  You can access the front-end of your Aegir install at:",
+==> valkyrie:             "=============================================================================",
+==> valkyrie:             "http://valkyrie.local/user/reset/1/1429111863/uAvC9VJx-l2CqhaufWg8ElCSFWzfdTiUTxJOQJ-pbz4/login",
+==> valkyrie:             "=============================================================================",
+==> valkyrie:             "/////////////////////////////////////////////////////////////////////////////"
+==> valkyrie:         ]
+==> valkyrie:     }
+==> valkyrie: }
+```
+
+* Use the login link as a one-time access to the now deployed aegir frontend, and reset the admin password immediate.  Don't forget to tick the check boxes for all the roles you would want the admin user to be able to play on the aegir server.
```

Comment:
Resetting the admin password isn't required. The password is random (and thus secure) at this point, and the session opened by clicking on the one-time password link will continue to grant access. If the session times-out, you need only run `drush @v uli` on the host to be logged in again.

Setting roles isn't required either, since the user is UID1.